api/tasks.go

```diff
@@ -374,9 +374,10 @@ type VolumeRequest struct {
 // VolumeMount represents the relationship between a destination path in a task
```

Comment:
Would need docs both for volume/volume_mount stanza and potentially mentioning it in the docker driver docs linking back to the stanza